Bridge/src/zeta.hpp

```diff
@@ -0,0 +1,109 @@
+#pragma once
+
+#include <functional>
+#include <thread>
+#include <vector>
+
+#include <ZetaRf.hpp>
+
+#include "frame.hpp"
+#include "thread_queue.hpp"
+#include "rf_config/si4455_revb1_bidir_fixed_crc_pre10_sync3MHP_pay8_433mhz_40kbps.hpp"
```

Comment:
Why choose this rf_config and not others? Would a comment explaining this be good here and/or what you can replace this with?